### PR TITLE
Lesson_13 Commit

### PR DESCRIPTION
**Pull request by ZarubeyAleksandr**

### CODE REVIEW

File: src/main/java/org/calculator/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    throw new ArithmeticException("Division by zero is not allowed.");
+                }
+                yield operand1 / operand2;
+            }
+            default -> throw new IllegalArgumentException("Invalid operator. Supported operators are +, -, x, /.");
+        };
+    }
+
+    public static void main(String[] args) {
+        //System.out.println(args.length);
```
Comment (mkharkhuta) [inline src/main/java/org/calculator/Calculator.java]: please remove redundant comments from the code

File: src/main/java/org/calculator/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+        try {
+            double operand1 = Double.parseDouble(args[0]);
+            double operand2 = Double.parseDouble(args[2]);
+            String operator = args[1];
+
+            double result = calculate(operand1, operator, operand2);
+            System.out.println("Result: " + result);
+        } catch (NumberFormatException e) {
+            System.out.println("Invalid input. Please enter valid numeric values.");
```
Comment (mkharkhuta) [inline src/main/java/org/calculator/Calculator.java]: it's better to user System.error.print() to print error messages

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```
Comment (mkharkhuta) [inline pom.xml]: this library is never used

**Review by mkharkhuta — COMMENTED** [review]
Good job 👍

File: pom.xml (hunk 3 above)
Comment (ZarubeyAleksandr) [reply pom.xml]: check

**Pull request merged**